src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -555,6 +564,30 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         menuItems.Append(addProfileItem);
     }
 
+    void MainPage::_SetupNavViewItems()

```

Comment:
For clarity, I might recommend we be _very specific_ about what this does.

`_moveXAMLParsedNavItemsIntoItemSource`

Reply:
In fact, we should make it a debug assert that it never fires twice